src/api/public/apidocs-new/paths/source_project_name_delete.yaml

```diff
@@ -0,0 +1,28 @@
+delete:
+  summary: Delete a specified project
+  description: |
+    Deletes a specified project and all the packages of this project
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - name: force
+      in: query
+      schema:
+        type: integer
+      description: |
+        If 1, the project is deleted even if the repositories of other
+        projects include a path to this project. After deletion, the path
+        in the other repository will start pointing towards 'deleted/standard' to
+        prevent the build and publishing
```

Comment:
I would try to adhere to the `Set to 1 ...` convention we're using on the documentation. Take a look [here](https://github.com/openSUSE/open-build-service/pull/13173/files#diff-1978fd1d2da315ced1d2163772f2fde44089cd0cdda101f8270d801122652348R23) so see an example.